app/src/main/res/values/strings.xml

```diff
@@ -197,7 +197,7 @@
     <string name="warning_password_encoding">As all letters of your password not supported in Latin-1 are converted to the same letter, it is recommended to make it compliantly unique.</string>
     <string name="warning_read_only">Grant KeePass DX write access to your SD card to save changes to your database.</string>
     <string name="warning_unmounted">Mount your SD card to create or load your database.</string>
-    <string name="warning_empty_password">Do you really want to use an empty string as your password?</string>
+    <string name="warning_empty_password">Do you really want no app protection?</string>
```

Comment:
No, an empty password does not mean that there will be no protection by other means, such as a file and key or some other method.